examples/pages/playgrounds/react-playgrounds/components/SubNav.tsx

```diff
@@ -2,14 +2,14 @@ import * as React from 'react';
 import classNames from 'classnames';
 
 interface IProps {
-    color?: 'lighter' | 'darker';
+    theme?: 'lighter' | 'darker';
```

Comment:
All components should use the same names for themes. Until now all components use `theme?: 'light' | 'dark';`. Can you use the same ones and map it to class names inside the component?